jetbrains-core/resources/telemetryOverride.json

```diff
@@ -317,6 +317,10 @@
                 },
                 {
                     "type": "cwsprChatConversationType"
+                },
```

Comment:
If these metrics are the same as VSC, should they be moved to common?

Reply:
No. At least no for now.

First of all, I made the change in toolkit common in v1.0.91 to add credential start url in Q telemetry, I tried to update the common dependency version. However, there are significant differences between 1.0.86 and 1.0.91. If I do this upgrade, I will have a lot of conflicts and missing definitions. It seems like the Amazon Q telemetry has diverged a lot from using common. Therefore, I have to override them.

Overriding the definition has to continue until all Q telemetry has been completely moved to common. THere is a ticket tracking that.

Reply:
By doing a diff between the definition override and common, you will find a huge diff and I won't be able to move them in this PR.

Reply:
The current consumed 1.0.86 common library has a old version of Q telemetry. For some reason, new Q telemetry events are not pushed to common library anymore, they are pushed as override in toolkit repo. This diverge happened weeks ago and the diff has been growing.

Due to this diff, when adding a new field in Q telemetry events, I cannot just add them to common repo (within a short amount of time). It takes too much due diligence to cherry pick and merge the override and common library. Instead I just push my changes to override json.

Reply:
Sounds good, moving to common can be a part of a separate PR, but we might want to do that soon or the diff will be larger! PS: not a req for this release